src/main/resources/config/application-local.properties

```diff
@@ -1,6 +1,7 @@
-
-alto.backend.corpus_name=synthetic
```

Comment:
do you want to include a template version of this, with the base_dir and source_text_dir settings left intentionally blank, and we can reference the template in the README instructions on how to run locally?